### PR TITLE
Weekly dose of fixes for issues reported by oss-fuzz

### PR DESCRIPTION
**Pull request by Jakuje**
##### Checklist

- [ ] PKCS#11 module is tested
- [ ] Windows minidriver is tested
- [ ] macOS tokend is tested

### CODE REVIEW

**Comment by vjardin** [comment]
LGTM (code review).

File: src/libopensc/card-cardos.c [hunk 1]
```diff
@@ -175,10 +175,10 @@ static int cardos_have_2048bit_package(sc_card_t *card)
 		return 0;
 
 	while (len != 0) {
-		p = sc_asn1_find_tag(card->ctx, p, len, 0xe1, &tlen);
+		pp = sc_asn1_find_tag(card->ctx, p, len, 0xe1, &tlen);
 		if (p == NULL)
```
Comment (frankmorgner) [inline src/libopensc/card-cardos.c:179]: `if (pp == NULL)`?

**Review by frankmorgner — CHANGES_REQUESTED** [review]

File: src/libopensc/card-cardos.c (hunk 1 above)
Comment (Jakuje) [reply src/libopensc/card-cardos.c:179]: Right. Good catch. Will fix.

**Comment by Jakuje** [comment]
@vjardin I see you working on iasecc now. The top commit is fixing one issue reported by oss-fuzz when the ACLs returns empty zero-length data, there was buffer overflow. Can you check my change is not breaking anything for you?

**Comment by Jakuje** [comment]
(I bundled up couple of new fixes for coverity too)

**Comment by Jakuje** [comment]
@frankmorgner did you manage to re-review this PR?

**Review by frankmorgner — APPROVED** [review]

**Pull request merged**